packages/autocomplete/src/create.ts

```diff
@@ -6,6 +6,7 @@ import { parseAutocomplete } from './parse'
 import type { AutocompleteOptions, ParsedAutocompleteTemplate, UnocssAutocomplete } from './types'
 import { searchAttrKey, searchUsageBoundary } from './utils'
 
+export const vscodeErrors = new Set<string>()
```

Comment:
`autocomplete` pkg was a pure package, it should not have any other redundant logic processing. You should match the error output when using it.

Reply:
Yeah I think it should not be VSCode specific, and ideally should be handled in the VS code's code